dashboard/app/models/teacher_feedback.rb

```diff
@@ -171,4 +167,17 @@ def increment_visit_count
     self.student_last_visited_at = now
     save
   end
+
+  private
+
+  def awaiting_teacher_review?(is_latest = false)
+    # only the latest feedback can be awaiting a teacher review
```

Comment:
Maybe expand this comment a bit to give some more context around _why_ this is the case?